src/components/CardArt.vue

```diff
@@ -3,15 +3,30 @@ import { computed, ref, watch } from "vue";
 import type { ScryfallCard } from "../models/scryfall-card";
 import { LoadingStatus } from "../store";
 import LoadingPulser from "./LoadingPulser.vue";
+import { getCardImages } from "../utils/card-image";
+import { pickRandomItem } from "../utils/math";
+
+type Props = {
+  /** The card to show art for. */
+  card: ScryfallCard;
+  /** Whether the next card is currently being loaded in. This activates a loading overlay. */
+  loading: LoadingStatus;
+};
 
-const props = defineProps<{ card: ScryfallCard; loadingNext: LoadingStatus }>();
+const props = defineProps<Props>();
 const imageUri = computed(() => {
-  if (props.card.image_uris) {
-    return props.card.image_uris.art_crop;
+  const imageUris = getCardImages(props.card);
+
+  if (imageUris.length === 0) {
+    return undefined;
   }
-  const randomIndex = Math.floor(Math.random() * props.card.card_faces.length);
-  const randomFace = props.card.card_faces[randomIndex];
-  return randomFace.image_uris?.art_crop;
+
+  if (imageUris.length === 1) {
+    return imageUris[0].art_crop;
+  }
```

Comment:
We don't have to do this, `pickRandomItem` will return the first item in an array of 1. In fact, we could make the whole function this, and it should behave the same:

```ts
const imageUri = computed(() => {
  const imageUris = getCardImages(props.card);
  const randomImageUri = pickRandomItem(imageUris);
  return randomImageUri?.art_crop;
});
```